### PR TITLE
Dev - worker optimization

### PR DESCRIPTION
**Pull request by jicata**

### CODE REVIEW

File: Open Judge System/Data/OJS.Data.Models/SubmissionsForProcessing.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿namespace OJS.Data.Models
+{
+    using System.ComponentModel.DataAnnotations.Schema;
+
+    [Table("SubmissionsForProcessing")]
+    public class SubmissionsForProcessing
```
Comment (YanaSlavcheva) [inline Open Judge System/Data/OJS.Data.Models/SubmissionsForProcessing.cs]: The class must be singular

File: Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/SubmissionsController.cs [hunk 2]
```diff
@@ -241,7 +243,7 @@ public ActionResult ConfirmDelete(int id)
             var submission = this.Data.Submissions
                 .All()
                 .FirstOrDefault(subm => subm.Id == id);
```
Comment (YanaSlavcheva) [inline Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/SubmissionsController.cs]: use` s` instead of `subm` imo

File: Open Judge System/Web/OJS.Web/OJS.Web.csproj [hunk 3]
```diff
@@ -35,6 +35,7 @@
     <UseGlobalApplicationHostFile />
     <NuGetPackageImportStamp>
     </NuGetPackageImportStamp>
+    <Use64BitIISExpress />
```
Comment (YanaSlavcheva) [inline Open Judge System/Web/OJS.Web/OJS.Web.csproj]: Is this change intended?

File: Open Judge System/Workers/OJS.Workers.LocalWorker/LocalWorkerService.cs [hunk 4]
```diff
@@ -83,9 +83,9 @@ protected override void OnStop()
         /// </summary>
         private void ResetAllProcessingSubmissions()
```
[the remark this answers is not in the document]
Comment (jicata) [reply Open Judge System/Workers/OJS.Workers.LocalWorker/LocalWorkerService.cs]: Done in [this commit]( https://github.com/SoftUni-Internal/OpenJudgeSystem/pull/105/commits/05bfed89617168201a1e794478af81b28af25d83) . Dunno why it's not being shown here.

File: Open Judge System/Data/OJS.Data/Repositories/Contracts/ISubmissionsForProcessingRepository.cs [hunk 5]
```diff
@@ -0,0 +1,12 @@
+﻿namespace OJS.Data.Repositories.Contracts
+{
+    using Data.Contracts;
+    using Models;
```
Comment (Innos) [inline Open Judge System/Data/OJS.Data/Repositories/Contracts/ISubmissionsForProcessingRepository.cs]: Split usings and use full name of the namespace.

File: Open Judge System/Data/OJS.Data/Repositories/SubmissionsForProcessingRepository.cs [hunk 6]
```diff
@@ -0,0 +1,48 @@
+﻿namespace OJS.Data.Repositories
+{
+    using System.Linq;
+    using Base;
```
Comment (Innos) [inline Open Judge System/Data/OJS.Data/Repositories/SubmissionsForProcessingRepository.cs]: Split usings and use full name of the namespace.

File: Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ParticipantsController.cs [hunk 7]
```diff
@@ -5,6 +5,7 @@
     using System.Collections.Generic;
     using System.Linq;
     using System.Web.Mvc;
+    using EntityFramework.Extensions;
```
Comment (Innos) [inline Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ParticipantsController.cs]: Split usings.

File: Open Judge System/Data/OJS.Data.Models/OJS.Data.Models.csproj [hunk 8]
```diff
@@ -79,6 +79,7 @@
     <Compile Include="ProblemResource.cs" />
     <Compile Include="Settings.cs" />
     <Compile Include="SourceCode.cs" />
+    <Compile Include="SubmissionsForProcessing.cs" />
```
Comment (YanaSlavcheva) [inline Open Judge System/Data/OJS.Data.Models/OJS.Data.Models.csproj]: You renamed the file to `SubmissionForProcessing`. You include the old name...
